### PR TITLE
Features/movie details page

### PR DESCRIPTION
**Pull request by DominikaBuczekSzkowron**
When Oleksander will finish his PeoplePage I going to change my MainTail to his PeopleTile. That's why my people section isn't look well yet.

### CODE REVIEW

File: src/Features/MovieDetailsPage/index.js [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+                  //to do: change MainTail to Oleksander version
+                  <MainTile
+                    id={id}
+                    poster={person.profile_path}
+                    title={person.name}
+                    subtitle={person.job}
+                    tags={[]}
+                  />
+                ))
+              : null}
```
Comment (Siedemus) [inline src/Features/MovieDetailsPage/index.js:136]: in the mobile version of our app person name and role/job are in random position, sometimes is sticked to right border sometimes is centered

**Review by Siedemus — COMMENTED** [review]
You made a really great job 🫡, but you still have something to fix. 👍👍👍

**Review by mileni67 — APPROVED** [review]

File: src/Common/MainTail/index.js
[the remark this answers is not in the document]
Comment (DominikaBuczekSzkowron) [reply src/Common/MainTail/index.js:1]: ok

File: src/Common/MainTail/styled.js
[the remark this answers is not in the document]
Comment (DominikaBuczekSzkowron) [reply src/Common/MainTail/styled.js:1]: ok

File: src/Common/Tile/index.js [hunk 2]
```diff
@@ -7,41 +6,64 @@ import {
...
+            {infoProduction.map(({ name }) => name).join(", ")}
+          </InfoLong>
+          <InfoShort>
+            {infoProduction.map(({ iso_3166_1 }) => iso_3166_1).join(", ")}
+          </InfoShort>
+        </Box>
+        <Box>
+          <AdditionInfo>Release date:</AdditionInfo>
+          <Info> {infoDate}</Info>
+        </Box>
```
[the remark this answers is not in the document]
Comment (DominikaBuczekSzkowron) [reply src/Common/Tile/index.js:52]: I'll add &nbsp; to my code but join(", ") works good here, I think.

File: src/Features/MovieDetailsPage/ScoresDetails/index.js [hunk 3]
```diff
@@ -0,0 +1,13 @@
+import { ReactComponent as Star } from "../../../Assets/Images/shape-star.svg";
+import { AllVotes, MaxScore, Score, Wrap, ScoresContainer } from "./styled";
+
+export const ScoresDetails = ({ data }) => (
+  <Wrap>
+    <ScoresContainer>
+      <Star />
+      <Score>{data.score}</Score>
+      <MaxScore>/10</MaxScore>
```
[the remark this answers is not in the document]
Comment (DominikaBuczekSzkowron) [reply src/Features/MovieDetailsPage/ScoresDetails/index.js:9]: good point! I used high-line: 130% - I hope it helped :)
fixed method is working now :)

File: src/Features/MovieDetailsPage/index.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (DominikaBuczekSzkowron) [reply src/Features/MovieDetailsPage/index.js:136]: ok

Comment (DominikaBuczekSzkowron) [reply src/Features/MovieDetailsPage/index.js:136]: I waiting for PeopleTail from Oleksander, than I correct my People section.

File: src/Features/MovieDetailsPage/index.js [hunk 4]
```diff
@@ -0,0 +1,145 @@
...
+  }, [id]);
+
+  const dispatch = useDispatch();
+
+  const moviesData = useSelector(selectMoviesList);
+
+  useEffect(() => {
+    dispatch(fetchMovies());
+    dispatch(fetchGenres());
+  }, [dispatch]);
```
[the remark this answers is not in the document]
Comment (DominikaBuczekSzkowron) [reply src/Features/MovieDetailsPage/index.js:76]: ok

**Pull request merged**